app/javascript/components/lease_ups/LeaseUpsTableContainer.js

```diff
@@ -56,33 +58,36 @@ class LeaseUpTableContainer extends React.Component {
 
   hideStatusModalAlert = () => this.updateStatusModal('showAlert', false)
 
-  leaseUpStatusChangeHandler = (applicationId, status) => {
+  leaseUpStatusChangeHandler = (applicationPreferenceId, applicationId, status) => {
+    console.log('leaseUpStatusChangeHandler', applicationPreferenceId, applicationId, status)
```

Comment:
looks like this got left in there. Also looks like you still have pre-commit hooks disabled